localization/english/wc_schemes_l_english.yml

```diff
@@ -0,0 +1,50 @@
+﻿l_english:
+
+ stasis_trap_title:0 "Stasis Trap:"
+
+ start_stasis_trap:0 "Stasis Trap #weak (Scheme)#!"
+ stasis_trap_scheme:0 "Stasis Trap Scheme"
+ stasis_trap_scheme_desc:0 "[stun|E] your target"
+ stasis_trap_scheme_action:0 "Stasis Trap"
+ stasis_trap_action:0 "Stasis Trap"
+ stasis_trap_desc:0 "I will [stun|E] [SCHEME.GetTarget.GetShortUINameNoTooltip]!"
+ stasis_trap_desc_general:0 "Attempt to [stun|E] an enemy."
+ scheme_interaction_tt_stasis_trap_approved:0 "[recipient.GetFirstName] will soon be [stunned|E]!"
```

Comment:
I don't approve lying in a tooltip.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/117035755-beaf2580-ad15-11eb-86a6-6d79ae45dd58.png)
![image](https://user-images.githubusercontent.com/46384992/117035783-ca9ae780-ad15-11eb-9f9f-ef34da38b59b.png)

</details>

Reply:
The english abduct equivalent is ` scheme_interaction_tt_abduct_approved:0 "[recipient.GetFirstName] will soon be my prisoner!"`

Reply:
> The english abduct equivalent is ` scheme_interaction_tt_abduct_approved:0 "[recipient.GetFirstName] will soon be my prisoner!"`

It doesn't change my opinion.